src/components/PostList.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Post from './Post';
+
+const PostList = ({ posts }) => (
+  posts.map(post => (
+    <Post post={post} key={post.id} />
```

Comment:
```suggestion
    <Post post={currentPost} key={post.id} />
```
three `post` in a row it's quite hard to understand